docs/GettingStartedDocs/README.md

```diff
@@ -39,7 +39,7 @@ release, start here for instructions to install and use the SDK package:
 - [Ubuntu 18.04 with SGX hardware](install_oe_sdk-Ubuntu_18.04.md)
 - [Ubuntu 16.04 with SGX hardware](install_oe_sdk-Ubuntu_16.04.md)
 - [Ubuntu 18.04 or 16.04 in simulation mode](install_oe_sdk-Simulation.md)
-- [Windows Server 2016 with SGX hardware](install_oe_sdk-Windows.md)
+- [Windows 10(1709 or higher), Windows Server 2016 or Windows Server 2019 on an Intel machine](install_oe_sdk-Windows.md)
```

Comment:
Just a comment: it seems confusing that Linux has separate pages for hardware vs simulation but Windows doesn't, why the difference?